server/src/app.js

```diff
@@ -19,7 +20,8 @@ app.use(helmet());
 app.use(morgan('combined'));
 
 app.use('/', homeRoutes);
-app.use('/', productRoutes)
+app.use('/', productsRoutes);
```

Comment:
I think it would be better, if you will add `/products` and `/clients` URL here and remove those prefixes from `*routes.js` files. It won't be duplicated in every endpoint for controller and will group those endpoints.

Reply:
Done